src/middlewares/validator.js

```diff
@@ -1,13 +1,44 @@
 const createError = require('http-errors')
 const Ajv = require('ajv')
 const ajvKeywords = require('ajv-keywords')
+const ajvLocalize = require('ajv-i18n')
 const {deepEqual} = require('assert')
+const parseFn = require(`negotiator/lib/language`)
 
 let ajv
 let previousConstructorOptions
-const defaults = {v5: true, $data: true, allErrors: true}
+const defaults = {
+  v5: true,
+  coerceTypes: 'array', // important for query string params
+  allErrors: true,
+  useDefaults: true,
+  $data: true // required for ajv-keywords
+}
+
+const chooseLanguage = (event) => {
+  const results = parseFn(event.headers['Accept-Language'], [
+    'en',
+    'ar',
+    'cz',
+    'de',
+    'es',
+    'fr',
+    'hu',
+    'it',
+    'ja',
+    'nb',
+    'pl',
+    'pt-BR',
+    'ru',
+    'sk',
+    'sv',
+    'zh'
+  ])
```

Comment:
Still not sure it's a good approach to list the languages here and not having any opportunity to configure the selection